androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/ui/theme/Colors.kt

```diff
@@ -4,4 +4,7 @@ import androidx.compose.ui.graphics.Color
 
 val primaryColor = Color(0xFFE4E2DE)
 val danger = Color(0xFFCD9A50)
-val colorIcon = Color(0xFF989383)
\ No newline at end of file
+val colorIcon = Color(0xFF989383)
+val lineOneCard = Color(0xFFF4D4E0)
+val lineTwoCard = Color(0xFFC2E0E2)
+val lineFourColor = Color(0xFFE1CCE3)
```

Comment:
Change to better names so it's more readable. We maybe want to reuse this colors somewhere else. For example: "rettsCardColor" or something like that.

Reply:
Fixed